client/rest-high-level/build.gradle

```diff
@@ -26,9 +26,9 @@ group = 'org.elasticsearch.client'
 dependencies {
   compile "org.elasticsearch:elasticsearch:${version}"
   compile "org.elasticsearch.client:rest:${version}"
-  compile "org.elasticsearch.plugin:parent-join-client:${version}"
-  compile "org.elasticsearch.plugin:aggs-matrix-stats-client:${version}"
```

Comment:
I am not sure. This goes against the notion of modules. If we ship them by default with core, we should also make them available by default with the client?